src/components/pages/roster-page/roster-page.tsx

```diff
@@ -79,6 +79,7 @@ import { Modal } from '../../../actions/modal.actions';
 import { UserSelector } from '../../../selectors/user.selector';
```

Comment:
There are a couple `useEffect()` calls in this file with warnings on the dependencies. Can you just double check those are as they should be and disable the warning with a comment if they look ok?